src/components/forms/inputs/InputBase.jsx

```diff
@@ -9,7 +9,8 @@ export default class InputBase extends React.Component {
         label: React.PropTypes.string,
         labelMsg: React.PropTypes.string,
         initialValue: React.PropTypes.string,
-        className: React.PropTypes.string
+        className: React.PropTypes.string,
+        maxLength: React.PropTypes.int
```

Comment:
Great job! One final thing I noticed is that I get a warning when navigating to Actions.
```
Warning: Failed prop type: TextInput: prop type `maxLength` is invalid; it must be a function, usually from React.PropTypes.
    in TextInput (created by InjectIntl(TextInput))
    in InjectIntl(TextInput) (created by ActionForm)
    in li (created by Form)
    in ul (created by Form)
    in form (created by Form)
    in Form (created by ActionForm)
    in ActionForm (created by EditActionPane)
    in div (created by EditActionPane)
    in div (created by EditActionPane)
    in EditActionPane (created by InjectIntl(EditActionPane))
    in InjectIntl(EditActionPane) (created by Connect(InjectIntl(EditActionPane)))
    in Connect(InjectIntl(EditActionPane)) (created by Section)
    in div (created by Section)
    in div (created by Section)
    in Section (created by InjectIntl(Section))
    in InjectIntl(Section) (created by App)
    in div (created by App)
    in div (created by App)
    in body (created by App)
    in html (created by App)
    in App (created by DragDropContext(App))
    in DragDropContext(App) (created by Connect(DragDropContext(App)))
    in Connect(DragDropContext(App))
    in IntlProvider (created by IntlReduxProvider)
    in Provider (created by IntlReduxProvider)
    in IntlReduxProvider (created by Connect(IntlReduxProvider))
    in Connect(IntlReduxProvider) warning.js:33:14
```

Javascript does not have an integer type (unfortunately imo), you have to use number here for the proptype to be properly interpreted.

Reply:
Ah! This should be fixed now.

Reply:
Great! It seems there are other issues related to the propTypes, not related to this pull request. I will create a new issue for those.